doc/rbd/rbd-persistent-cache.rst

```diff
@@ -7,14 +7,14 @@
 Shared, Read-only Parent Image Cache
 ====================================
 
-`Cloned RBD images`_ from a parent usually only modify a small portion of
-the image. For example, in a VDI workload, the VMs are cloned from the same
-base image and initially only differ by hostname and IP address. During the
-booting stage, all of these VMs would re-read portions of duplicate parent
-image data from the RADOS cluster. If we have a local cache of the parent
-image, this will help to speed up the read process on one host, as well as
-to save the client to cluster network traffic.
-RBD shared read-only parent image cache requires explicitly enabling in
+`Cloned RBD images`_ usually only modify a small portion of the parent
```

Comment:
s/usually only modify/usually modify only/

This is super nitpicky, and I won't be upset if it's too much shoe leather to get it in, but anyway I'm saying it.

Reply:
You aren't wrong.  I debated how to rephrase that at first, I knew for sure it needed to be changed.  I've also changed `portion` to `fraction` since we aren't talking about garlic fries.  But of course we should.